tests/framework/microvm.py

```diff
@@ -552,6 +589,43 @@ def start(self):
         response = self.actions.put(action_type='InstanceStart')
         assert self._api_session.is_status_no_content(response.status_code)
 
+    def pause_to_snapshot(self,
+                          mem_file_path=None,
+                          snapshot_path=None,
+                          diff=False):
+        """Pauses the microVM, and creates snapshot.
+
+        This function validates that the microVM pauses successfully and
+        returns the name of the snapshot files.
```

Comment:
Why return the names of the snapshot files?

Reply:
Yup, no need to return anything. This is likely a leftover.

Reply:
Comment needs to be updated